### DISCUSSION TITLE
workflow priority value

### DISCUSSION THREAD

**Discussion by rickyschechter** [Netflix/conductor#2453] [Q&A]
Looking at: https://netflix.github.io/conductor/gettingstarted/startworkflow/#start-workflow-request, does the "priority" parameter supposedly "prioritize" same task names across workflows to "skip" place in queue for tasks of a higher priority workflow? My initial test did not yield any difference in the order of tasks I was polling no matter the priority value I would use for the workflow I created. Please advise, thanks!

**Comment by kishorebanala** [comment]
@rickyschechter Can you provide more details about the Conductor set-up, especially which implementation of queues you are using please? I believe folks have reported issues with priority in Dyno-queues earlier. Can you also check if the priority is being supported by the queuing layer you are using.

**Reply by rickyschechter** [reply]
As I understand, we use basic Dynomite and ElasticSearch per default configurations. Would the following config response give you what you are asking? I am more a developer than the devops that set it up.

Using version 2.30.4

http://localhost:5000/api/admin/config

{
  "jetty.git.hash": "b1e6b55512e008f7fbdf1cbea4ff8a6446d1073b",
  "workflow.repairservice.enabled": "true",
  "io.netty.noUnsafe": "true",
  "conductor.jetty.server.enabled": "true",
  "io.netty.noKeySetOptimization": "true",
  "buildDate": "2021-01-12_01:13:20",
  "io.netty.recycler.maxCapacityPerThread": "0",
  "conductor.grpc.server.enabled": "false",
  "version": "0.1.0-dev.0.uncommitted",
  "queues.dynomite.nonQuorum.port": "22122",
  "workflow.elasticsearch.url": "stage-conductor-elasticsearch:9300",
  "workflow.namespace.queue.prefix": "conductor_queues",
  "user.timezone": "GMT",
  "workflow.dynomite.cluster.name": "dyno1",
  "sun.nio.ch.bugLevel": "",
  "workflow.dynomite.cluster.hosts": "stage-conductor-dynomite:8102:us-east-1c",
  "workflow.elasticsearch.instanceType": "external",
  "db": "dynomite",
  "queues.dynomite.threads": "10",
  "workflow.namespace.prefix": "conductor",
  "workflow.elasticsearch.index.name": "conductor"
}

Hopefully it helps, thanks!
